docs/sources/clients/_index.md

```diff
@@ -76,3 +76,8 @@ when using or writing a third-party client.
 - [mjaron-tinyloki-java](https://github.com/mjfryc/mjaron-tinyloki-java) (Java)
 - [LokiLogger.jl](https://github.com/JuliaLogging/LokiLogger.jl) (Julia)
 - [winston-loki](https://github.com/JaniAnttonen/winston-loki) (JS)
+
+# Loki Clients for reading data
+
```

Comment:
Could we add a sentence here, something like  "Grafana Loki supports the following clients for reading log data."  ?